components/OurServices/index.tsx

```diff
@@ -19,8 +19,9 @@ const OurServices = () => {
       image: '/images/mobiledevelopment.svg',
     },
   ];
+  const service = useRef<HTMLInputElement>(null);
   return (
-    <div className="w-full">
+    <div className="w-full" ref={service} id="service">
```

Comment:
Why ref and id?

Reply:
for moving a perticular section within same page... like services section